### PR TITLE
refactor: clean up code structure of cancel order endpoints

### PR DESCRIPTION
**Pull request by WilliamHawkingStephen**
## What does this MR do?

- improve cancel and cancel all order endpoints

## Check list  

- [x] All Merge Request pipelines should be passed.
- [x] Make sure all the changes are necessary and correct.
- [x] Make sure no needed change missing (all necessary changes are made).
- [x] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [ ] All necessary unit tests are written.
- [x] Critical changes should be tested locally (or on staging).

### CODE REVIEW

File: src/presentation/http/controllers/order.controller.ts [hunk 1]
```diff
@@ -72,29 +72,32 @@ export class OrderController extends BaseController {
...
       network,
     };
-    const order_ids = await OrderController.processOrderService.cancelAll(
-      input,
-    );
-    res.send({ success: true, data: { order_ids } });
+
+    const data = await OrderController.processOrderService.cancelAll(input);
+
+    res.send({ success: true, data });
```
Comment (GreatGaryCooper) [inline src/presentation/http/controllers/order.controller.ts:100]: This causes the tests to fail, right now the response is an object that has a `data` field which is an array of numbers, but previously `data` field was an object that had a field called `order_ids` that was an array of numbers.

File: src/presentation/http/controllers/order.controller.ts [hunk 2]
```diff
@@ -72,29 +72,32 @@ export class OrderController extends BaseController {
   public async cancel(req: Request, res: Response, next: NextFunction) {
     const { id } = req.body;
     const { user_address } = req.userInfo;
-    const input: IProcessOrderService.cancelInput = {
+
+    const input = {
```
Comment (GreatGaryCooper) [inline src/presentation/http/controllers/order.controller.ts:76]: Also, we don't use `input` anywhere else in this function, so we can just pass `id` and `userAddress` directly to `cancel` method.

File: src/presentation/http/controllers/order.controller.ts [hunk 3]
```diff
@@ -72,29 +72,32 @@ export class OrderController extends BaseController {
...
+    res.send({ success: true, data });
     next();
   }
 
   public async cancelAll(req: Request, res: Response, next: NextFunction) {
     const { market, side } = req.body;
     const { user_address, network } = req.userInfo;
-    const input: IProcessOrderService.cancelAllInput = {
+
+    const input = {
```
Comment (GreatGaryCooper) [inline src/presentation/http/controllers/order.controller.ts:91]: Same applies here.

**Comment by hypersolo** [comment]
as we discussed before all our endpoint responses should have {success: boolean; data: object} interface.
by merging this pr, this convention wouldn't be followed!

**Review by GreatGaryCooper — APPROVED** [review]

**Pull request closed without merging**